specs/specs.js

```diff
@@ -15,8 +15,9 @@ var expect = require('chai').expect,
 
 describe('Scan Fn', function () {
 	it('should scan using a string', function (done) {
-		var stream = scan({ term: '@import', fn: function (match) {
+		var stream = scan({ term: '@import', fn: function (match, matchedFile) {
 				expect(match).to.equal('@import');
+				expect(matchedFile).to.equal(file);
```

Comment:
will likely fail after the file.clone change. compare a few properties of the files instead